client-app/core/api/graphql/fragments/allCurrencyFields.graphql

```diff
@@ -0,0 +1,8 @@
+fragment allCurrencyFields on CurrencyType {
+  code
+  symbol
+  exchangeRate
```

Comment:
We don't use nor `exchangeRate`, neither `cultureName` or `englishName`

Reply:
keep for compatibility with old /context method for now